tests/builders/test_builders.py

```diff
@@ -138,3 +138,15 @@ def test_file_count():
         configuration=config,
     )
     cmd.build()
+
+def test_file_regex():
+    """This test buildspec using status check with 'file_regex'"""
```

Comment:
Please change the comment her to match with the correct yml file name that will be linked for test.